addons/udes_sale_stock/models/sale_order.py

```diff
@@ -103,21 +103,51 @@ def get_available_quantity(self, product, locations):
     def cancel_orders_without_availability(self):
         """ From the current list of unconfirmed SO lines, cancel lines that
         cannot be fulfilled with current stock holding """
```

Comment:
Nit: docstring doesn't mention that the method returns the lines the unfulfillable lines.